### PR TITLE
External oauth

### PR DESCRIPTION
**Pull request by matthewshaver**
## What are you changing in this pull request and why?

Taking the External OAuth docs from unlisted to the site:
Adds the doc to the sidebar
Removes beta flags
Removes unlisted flags
Editorial updates to the doc

## Checklist
- [ ] I have reviewed the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] The topic I'm writing about is for specific dbt version(s) and I have versioned it according to the [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and/or [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content) guidelines.
- [ ] I have added checklist item(s) to this list for anything anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

### CODE REVIEW

File: website/docs/docs/cloud/manage-access/external-oauth.md [hunk 1]
```diff
@@ -23,7 +20,7 @@ dbt Cloud Enterprise supports [external OAuth authentication](https://docs.snow
 
 ## Getting started
```
Comment (runleonarun) [inline website/docs/docs/cloud/manage-access/external-oauth.md:21]: DO we need to link to the snowflake doc in the above paragraph?

File: website/docs/docs/cloud/manage-access/external-oauth.md [hunk 2]
```diff
@@ -187,68 +196,74 @@ In your Azure portal, open the **Entra ID** and click **App registrations** from
...
+1. From the Client ID 
+app in Entra ID, click **Endpoints** and open the **Federation metadata document** in a new tab.
+   - The **entity ID** on this page maps to the `external_oauth_issuer` field in the Snowflake config.
 2. Back on the list of endpoints, open the **OpenID Connect metadata document** in a new tab.
-    - The **jwks_uri** field maps to the `external_oauth_jws_keys_url` field in Snowflake. 
+   - The **jwks_uri** field maps to the `external_oauth_jws_keys_url` field in Snowflake.
 3. Navigate to the resource server in previous steps.
-    - The **Application ID URI** maps to teh `external_oauth_audience_list` field in Snowflake.
-4. Run the configurations. Be sure the admin who created the Microsoft apps is also a user in Snowflake, or the configuration will fail. 
+   - The **Application ID URI** maps to teh `external_oauth_audience_list` field in Snowflake.
```
Comment (runleonarun) [inline website/docs/docs/cloud/manage-access/external-oauth.md]:
```suggestion
   - The **Application ID URI** maps to the `external_oauth_audience_list` field in Snowflake.
```

**Review by runleonarun — APPROVED** [review]
Approving with some non blocking feedback and typo!

File: website/docs/docs/cloud/manage-access/external-oauth.md (hunk 1 above)
Comment (matthewshaver) [reply website/docs/docs/cloud/manage-access/external-oauth.md:21]: Good call!

File: website/docs/docs/cloud/manage-access/external-oauth.md [hunk 3]
```diff
@@ -1,20 +1,17 @@
...
-# Set up external Oauth <Lifecycle status="beta, enteprise" />
+# Set up external OAuth <Lifecycle status="enteprise" />
 
-:::note Beta feature
+:::note 
 
-External OAuth for authentication is available in a limited beta. If you are interested in joining the beta, please contact your account manager. 
-
-This feature is currently only available for the Okta and Entra ID identity providers and Snowflake connections. Only available to Enterprise accounts.
+This feature is currently only available for the Okta and Entra ID identity providers and Snowflake connections.
```
Comment (matthewshaver) [inline website/docs/docs/cloud/manage-access/external-oauth.md]:
```suggestion
This feature is currently only available for the Okta and Entra ID identity providers and [Snowflake connections](/docs/cloud/connect-data-platform/connect-snowflake).
```

**Pull request merged**